src/Apps/Artwork/Components/ArtistInfo.tsx

```diff
@@ -148,23 +149,7 @@ export const ArtistInfoQueryRenderer = ({ artistID }: { artistID: string }) => {
                 }
               }
             `}
-            render={({ props }) => {
-              if (props) {
-                return (
-                  <ArtistInfoFragmentContainer
-                    artist={props.artist as any}
-                    user={user}
-                    mediator={mediator}
-                  />
-                )
-              } else {
-                return (
-                  <Box width="100%" height="100px" position="relative">
-                    <Spinner />
-                  </Box>
-                )
-              }
-            }}
+            render={renderWithLoadProgress(ArtistInfoFragmentContainer as any)}
```

Comment:
@alloy - updating here per your pointer. This def looks way simpler in this class! But to get it to compile I had to cast ArtistInfoFragmentContainer as any. Is that acceptable or am I missing something here?

Reply:
Huh, interesting, I’m pretty sure I tested this to work. I’ll pull this down tomorrow and have a look.

Reply:
Seems to work fine for me. Maybe your vscode needs a ‘Reload Window’?

<img width="562" alt="screen shot 2018-10-11 at 01 36 35" src="https://user-images.githubusercontent.com/2320/46771990-3780a900-ccf6-11e8-9f91-4280dd70b1c9.png">